junit-test/run-tests.py

```diff
@@ -0,0 +1,53 @@
+from subprocess import call
+
+tests = [
+    ("testOnly org.scalajs.junit.success.*", ['n']),
+    ("testOnly org.scalajs.junit.success.*", ['n', 'v']),
+    ("testOnly org.scalajs.junit.success.*", []),
+    ("testOnly org.scalajs.junit.success.*", ['v']),
+    ("testOnly org.scalajs.junit.success.*", ['q']),
+    ("testOnly org.scalajs.junit.success.*", ['v', 'q']),
+    ("testOnly org.scalajs.junit.namedecode.*", ['v']),
+    ("testOnly org.scalajs.junit.namedecode.*", ['v', 'n']),
+    ("testOnly org.scalajs.junit.namedecode.*", ['v', 's']),
+    ("testOnly org.scalajs.junit.namedecode.*", ['v', 's', 'n']),
+    ("checkJUnitTestFailuresOutput", ['n']),
+    ("checkJUnitTestFailuresOutput", ['n', 'a']),
+    ("checkJUnitTestFailuresOutput", ['n', 'v']),
+    ("checkJUnitTestFailuresOutput", ['n', 'v', 'a']),
+    ("checkJUnitTestFailuresOutput", ['n', 'v', 'c']),
+    ("checkJUnitTestFailuresOutput", ['n', 'v', 'c', 'a']),
+    ("checkJUnitTestFailuresOutput", []),
+    ("checkJUnitTestFailuresOutput", ['q']),
+    ("checkJUnitTestFailuresOutput", ['a']),
+    ("checkJUnitTestFailuresOutput", ['v']),
+    ("checkJUnitTestFailuresOutput", ['v', 'q']),
+    ("checkJUnitTestFailuresOutput", ['v', 'a']),
+    ("checkJUnitTestFailuresOutput", ['v', 'c']),
+]
+
+jUnitTestJS = "jUnitTestJS"
+jUnitTestJVM = "jUnitTestJVM"
+
+cmd = ['sbt']
+
+def setFlagsInScope(flags, scope):
+    args = map(lambda x: '"-' + x + '"', flags)
+    cmd.append(
+        "set inScope(ThisScope in " + scope +")(jUnitTestArgs := Seq(" + \
+        ', '.join(args) + "))")
```

Comment:
`\` is not necessary inside the `()` of the call.
Also, consider using a `%`-formatted string:
```python
    cmd.append(
        "set inScope(ThisScope in %s)(jUnitTestArgs := Seq(%s))" %
        (scope, ", ".join(args)))
```